app/views/users/show.html.erb

```diff
@@ -151,6 +151,8 @@
       </button>
       <div id="user-profile-dropdownmenu" class="profile-dropdownmenu">
         <a href="/report-abuse?url=https://dev.to/<%= @user.username %>">Report Abuse</a>
+        <button id="user-profile-dropdownmenu-block-button" data-profile-user-id="<%= @user.id %>" class="block-button" type="button">Block</button>
+        <%= javascript_pack_tag "blockButton", defer: true %>
```

Comment:
I'm wondering if this pack would be better named something like `userProfileDropbdownMenu` so that the next logical functionality could be added to the file rather than having to rename it later or add another pack?

The functionality within the file could remain the same, but this seems simpler if we may wind up tacking more functionality here.

Reply:
That makes sense, will update 👍